public/app/features/explore/LiveLogs.tsx

```diff
@@ -108,8 +113,16 @@ class LiveLogs extends PureComponent<Props, State> {
         </div>
         <div className={cx([styles.logsRowsIndicator])}>
           <span>
-            Last line received: <ElapsedTime renderCount={renderCount} humanize={true} /> ago
+            Last line received: <ElapsedTime resetKey={this.props.logsResult} humanize={true} /> ago
```

Comment:
Changed a bit how ElapsedTime timer is reset so that we do not need counter here and we save one additional rerender just for the counter update.